### ISSUE TITLE
version confusion

### ISSUE THREAD

**Issue by mikeatcityuni** [tonyjbutler/moodle-mod_aspirelist#2]
Hi Tony, just updated this from Moodle plugins database to latest version for 2.8 (2015012102) and there are much less admin settings (e.g. no includechildcodes setting) than before (2014103001). have I got wrong version? Can I use 2015052100 for M2.8?

**Comment by tonyjbutler** [comment]
Hi Mike,

2015012102 is indeed the correct latest version for Moodle 2.8. I've just downloaded the zip file (https://moodle.org/plugins/download.php/7787/mod_aspirelist_moodle28_2015012102.zip) from the plugins directory as a sanity check, and nothing has been removed. The includechildcodes setting is still there in settings.php.

2015052100 is a new version for Moodle 2.9 and doesn't support 2.8.

Cheers,
Tony
